src/router/index.jsx

```diff
@@ -65,7 +67,11 @@ function getClientRoutes() {
     },
     {
       path: '/order-status',
-      element: <CustomerOrderStatus />,
+      element: (
+        <ChakraProvider theme={theme}>
+          <CustomerOrderStatus />
+        </ChakraProvider>
```

Comment:
it didn't work in your case because ChakraProvider gets overriden on ClientLayout component - please remove it both from client and admin

Reply:
ok, i removed those chakra provider wrappers around layouts, but in case it breaks smth i dont know:) i didnt work on layouts, maybe it was needed for smth. looks alright so far